tests/Kernel/TimelineTest.php

```diff
@@ -0,0 +1,218 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Kernel;
+
+use Drupal\Core\Entity\Entity\EntityViewDisplay;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\filter\Entity\FilterFormat;
+use Drupal\node\Entity\Node;
+use Drupal\node\Entity\NodeType;
+use Drupal\Tests\rdf_entity\Traits\RdfDatabaseConnectionTrait;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Tests the timeline field type definition.
+ */
+class TimelineTest extends AbstractKernelTestBase {
+
+  use RdfDatabaseConnectionTrait;
+
+  /**
+   * The node storage.
+   *
+   * @var \Drupal\node\NodeStorageInterface
+   */
+  protected $nodeStorage;
+
+  /**
+   * The node view builder.
+   *
+   * @var \Drupal\Core\Entity\EntityViewBuilderInterface
+   */
+  protected $nodeViewBuilder;
+
+  /**
+   * A field storage to use in this test class.
+   *
+   * @var \Drupal\field\Entity\FieldStorageConfig
+   */
+  protected $fieldStorage;
```

Comment:
Is there any reason why we store the fieldStorage in a parameter?

Reply:
Yes, because we need to reuse it. Otherwise it would not be needed indeed.